autoria.py

```diff
@@ -399,11 +399,26 @@ def select_item(item_to_select: str, items_list: dict) -> int:
             needed item (category, bodystyle, mark etc.) identifyer.
         """
         if item_to_select is not None and items_list is not None:
+            result = None
             for item in items_list:
-                if fnmatch(item['name'], '*' + item_to_select + '*'):
-                    return item['value']
-        else:
-            return None
+                if fnmatch(item['name'], item_to_select):
+                    result = item['value']
+                    print(result)
+                    return result
+            if result is None:
+                for item in items_list:
```

Comment:
OK, I see, just for the record, it is possible to have one loop having something like this:

```
starts_with_matches = []
contains_matches = []

for item in items_list:
    if fnmatch(item['name'], item_to_select):
        # found exact match, return it
        return item['value']
    if fnmatch(item['name'], '{}*'.format(item_to_select)):
        starts_with_matches.append(item['value'])
    if fnmatch(item['name'], '*{}*'.format(item_to_select)):
        contains_matches.append(item['value'])

if starts_with_matches:
    return starts_with_matches[0]

if contains_matches:
    return contains_matches[0]
```

But probably, it is not worth changing unless we have a real performance problem due to three loops (which I doubt we have).